agent_build/package_builders.py

```diff
@@ -1156,3 +1179,21 @@ class DockerApiPackageBuilder(ContainerPackageBuilder):
         base_image_deployment_step_cls=deployments.BuildAlpineDockerBaseImageStep,
     )
 )
+
+# Those builds for a temporary testing of the "stop_agent_on_failure" option for the k8s image build.
+K8S_CONTAINER_RESTART_AGENT_ON_MONITOR_DEATH_DEBIAN = K8sRestartAgentOnMonitorsDeath(
+    name="k8s-restart-agent-on-monitor-death-debian",
+    config_path=_CONFIGS_PATH / "k8s-config",
+    base_image_deployment_step_cls=deployments.BuildDebianDockerBaseImageStep,
+    additional_config_paths=[
```

Comment:
Ah, OK, you added support for additional config paths - I thought you will just copy over the contents from https://github.com/scalyr/scalyr-agent-2/tree/master/docker/k8s-config/agent.d and then modify docker.json, but this approach works as well :)